packages/nuemark/src/render-tag.js

```diff
@@ -142,9 +153,21 @@ export function renderIcon(name, symbol, icon_dir) {
 
 export function renderTag(tag, opts = {}) {
   const tags = { ...TAGS, ...opts.tags }
-  const fn = tags[tag.name || 'block']
+  const fn = tags[!tag.is_block && tag.name || 'block']
+
+  if (!fn) {
+    // native html tags
+    if (HTML_TAGS.includes(tag.name)) {
+      // inline / block without blocks, but with '_' data
+      if (tag.is_inline || (!tag.blocks?.length && tag.data?._)) return elem(tag.name, tag.attr, renderInline(tag.data?._, opts))
```

Comment:
```suggestion
      if (tag.is_inline || !tag.blocks?.length) return elem(tag.name, tag.attr, renderInline(tag.data?._, opts))
```
Could do it like this. If there's no block content, it doesn't matter if the _ data is set or not.